front/app/containers/HomePage/index.tsx

```diff
@@ -41,8 +37,6 @@ const HomePage = () => {
     }
   }, [pressedLetterAKey, userHasAdminAccess]);
 
-  if (!homepageLayout) return <Spinner />;
```

Comment:
I'm not exactly sure what's causing the spinner to go on infinitely (why the component doesn't re-render when homepageLayout updates 🤔) but this is the spinner that we're seeing. Also, we're not using homepageLayout in this component, so we can remove it. I suspect this will resolve the issue.